src/gpl/src/graphics.cpp

```diff
@@ -534,4 +534,53 @@ bool Graphics::guiActive()
   return gui::Gui::enabled();
 }
 
+void Graphics::addFrameLabel(gui::Gui* gui,
+                             const odb::Rect& bbox,
+                             const std::string& label,
+                             const std::string& label_name,
+                             int image_width_px)
+{
+  int label_x = bbox.xMin() + 300;
+  int label_y = bbox.yMin() + 300;
+
+  gui::Painter::Color color = gui::Painter::stringToColor("yellow", logger_);
+  gui::Painter::Anchor anchor
+      = gui::Painter::stringToAnchor("bottom left", logger_);
```

Comment:
you could just reference then as `gui::Painter::yellow` and `gui::Painter::BOTTOM_LEFT` that avoid the string lookup which is only needed in the tcl interface